src/cmdstan-guide/generate_quantities_config.qmd

```diff
@@ -1,21 +1,75 @@
+Instead, you write a modified version of the original Stan program
+and add a generated quantities block or modify the existing one
+which specifies how to compute the new quantities of interest.
+Running the `generate_quantities` method on the new program
+together with sampler outputs (i.e., a set of draws)
+from the fitted model runs the generated quantities block
+of the new program using the the existing sample by plugging
+in the per-draw parameter estimates for the computations in
+the generated quantities block.
 
 This method requires sub-argument `fitted_params` which takes as its value
-an existing Stan CSV file that contains a sample from an equivalent model,
-i.e., a model with the same parameters, transformed parameters, and model blocks,
+an existing [Stan CSV](stan_csv_apdx.qmd) file that contains a parameter values
+from an equivalent model, i.e., a model with the same parameters and transformed parameters blocks,
 conditioned on the same data.
 
+The [generated quantities block](https://mc-stan.org/docs/reference-manual/blocks.html#generated-quantities)
+computes *quantities of interest* (QOIs) based on the data,
+transformed data, parameters, and transformed parameters.
+It can be used to:
+
+-  generate simulated data for model testing by forward sampling
+-  generate predictions for new data
+-  calculate posterior event probabilities, including multiple
+   comparisons, sign tests, etc.
+-  calculating posterior expectations
+-  transform parameters for reporting
+-  apply full Bayesian decision theory
+-  calculate log likelihoods, deviances, etc. for model comparison
+
+
+For an overview of the uses of this feature, see the Stan User's Guide section on
+[Stand-alone generated quantities and ongoing prediction](https://mc-stan.org/docs/stan-users-guide/posterior-prediction.html#stand-alone-generated-quantities-and-ongoing-prediction).
+
+
+## Example
+
+To illustrate how this works we use the `generate_quantities` method
+to do posterior predictive checks using the estimate of `theta` given
+the example bernoulli model and data, following the
+[posterior predictive simulation](https://mc-stan.org/docs/stan-users-guide/posterior-prediction.html#posterior-predictive-simulation-in-stan)
+procedure in the Stan User's Guide.
+
+We write a program `bernoulli_ppc.stan` which contains
+the following generated quantities block, with comments
+to explain the procedure:
+```stan
+generated quantities {
+  real<lower=0, upper=1> theta_rep;
+  array[N] int y_sim;
+  // use current estimate of theta to generate new sample
+  for (n in 1:N) {
+    y_sim[n] = bernoulli_rng(theta);
+  }
+  // estimate theta_rep from new sample
+  theta_rep = sum(y_sim) * 1.0 / N;
+}
+```
+The rest of the program is the same as in `bernoulli.stan`.
+
+The `generate_method` requires the sub-argument `fitted_params`
+which takes as its value the name of a Stan CSV file.
+The per-draw parameter estimates from the `fitted_params` file will
```

Comment:
The rows of the fitted_params file are not estimates---they're just draws.  Estimates are based on averaging over the rows (or taking a median).  So just saw "The draws in the `fitted_params` file..."

Reply:
You can also run `generate_quantities` with the `fitted_params` file from something like optimization (though doing so would be an odd choice, certainly).

I've swapped the word from "estimates" to "values"